festim/hydrogen_transport_problem.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+
+class HydrogenTransportProblem:
+    """
+    Hydrogen Transport Problem.
+    Used internally in festim.Model
+
+    Args:
+        geometry (festim.Geometry): the geometry of the model (mesh, function
+            spaces and subdomains)
+
+    Attributes:
+        geometry (festim.Geometry): the geometry of the model (mesh, function
+            spaces and subdomains)
+
+    """
+
+    def __init__(
+        self,
+        geometry=None,
+        species=[],
+        temperature=None,
+        boundary_conditions=[],
+        solver_parameters=None,
+        exports=[],
+    ) -> None:
+        self.geometry = geometry
+        self.species = species
+        self.temperature = temperature
+        self.boundary_conditions = boundary_conditions
+        self.solver_parameters = solver_parameters
+        self.exports = exports
```

Comment:
Similar to comments above, I have realised that measures are attributes of the `Model` not `Geometry.`